server/sonar-db-dao/src/test/java/org/sonar/db/organization/OrganizationDaoTest.java

```diff
@@ -1020,7 +1020,7 @@ private OrganizationDto copyOf(OrganizationDto organizationDto) {
       .setName(organizationDto.getName())
       .setDescription(organizationDto.getDescription())
       .setUrl(organizationDto.getUrl())
-      // TODO : .setDefaultQualityGateUuid(dbClient.qualityGateDao().selectBuiltIn(dbSession).getId().toString())
+      .setDefaultQualityGateUuid(organizationDto.getDefaultQualityGateUuid())
```

Comment:
As the commit is only about existing quality gate, nothing should has been done on new one